disk/datadog_checks/disk/disk.py

```diff
@@ -78,10 +78,14 @@ def check(self, instance):
 
             # Get disk metrics here to be able to exclude on total usage
             try:
-                disk_usage = timeout(5)(psutil.disk_usage)(part.mountpoint)
+                self.log.debug("Check %s", part.mountpoint)
```

Comment:
```suggestion
                self.log.debug("Get disk usage data for mountpoint %s", part.mountpoint)
```

If that debug log is intended, the message could be a bit more explicit IMO.